README.md

```diff
@@ -130,6 +130,28 @@ There is also a fourth directory, `/etc/prometheus/alerting/generated`, where th
 
 If you want to disable the pre-processing and use entirely your own ruleset, set the environment variable `DISABLE_ALERTS_PREPARE=true`.
 
+#### Cluster Monitor checker results
+
+If running with the Couchbase Cluster Monitor enabled, it will output the status of its checkers to Prometheus like the following:
+
+```
+# HELP multimanager_cluster_checker_status Checker results for cluster level checkers
+# TYPE multimanager_cluster_checker_status gauge
+multimanager_cluster_checker_status{cluster="961b7464aca17a7db150fb0e44be5849",name="singleOrTwoNodeCluster"} 1
+# HELP multimanager_node_checker_status Checker results for node level checkers
+# TYPE multimanager_node_checker_status gauge
+multimanager_node_checker_status{cluster="961b7464aca17a7db150fb0e44be5849",name="nonGABuild",node="47a45d39583947823ac9866dce2a74b2"} 0
+multimanager_node_checker_status{cluster="961b7464aca17a7db150fb0e44be5849",name="oneServicePerNode",node="47a45d39583947823ac9866dce2a74b2"} 1
+multimanager_node_checker_status{cluster="961b7464aca17a7db150fb0e44be5849",name="supportedVersion",node="47a45d39583947823ac9866dce2a74b2"} 0
+```
+
+The metric values are mapped as follows:
```

Comment:
I think we need a bit more detail here as it's not clear to me with a single word what each covers. I'd expect an increase in value to increase in priority as well but that may not be an obvious mapping for Grafana.

Reply:
As far as I'm aware the mapping is arbitrary - and I'm a bit hesitant about changing it, because it may be used in other places (e.g. stored in the alert history). Having said that, we can probably get away with it before 0.1 - what do you think?

Reply:
Yeah I would change it before we get too far along. Separate JIRA or just another PR on this JIRA though is fine to close this one out.